### PR TITLE
add rocky linux optimized for gcp

### PR DESCRIPTION
**Pull request by hopkiw**
add rocky-optimized-gcp-8; amend rollout rate for rhui

### CODE REVIEW

File: concourse/pipelines/linux-image-build.jsonnet [hunk 1]
```diff
@@ -19,7 +19,7 @@ local rhuiimgbuildtask = imgbuildtask {
 
   run+: {
     // Prepend, as the workflow must be the last arg. Daisy is picky.
-    args: ['-gcs_path=gs://rhel-infra-daisy-bkt/'] + super.args,
+    args: ['-gcs_path=gs://rhel-infra-daisy-bkt/', '-rollout_rate=0'] + super.args,
```
Comment (Linskeyd) [inline concourse/pipelines/linux-image-build.jsonnet:22]: Should this one be a part of this PR? I may be mistaken, but it seems unrelated to the Rocky image addition.

Comment (hopkiw) [reply concourse/pipelines/linux-image-build.jsonnet:22]: yeah this is for rhui!
